### PR TITLE
Feat add expired counts to unit cards

### PR DESCRIPTION
**Pull request by JBR90**
# How to review this PR

_Leave this text block for the reviewer_

- Check [component hierarchy](https://miro.com/app/board/uXjVNnKBgyk=/?share_link_id=59445593794) is followed correctly
- Check the design [Heuristics](https://components.thenational.academy/?path=/docs/docs-howtodesigncomponents--docs#heuristics-for-component-design) have been followed
- Check [naming conventions](https://components.thenational.academy/?path=/docs/docs-namingconventions--docs) have been applied
- Check for these gotchyas:
  - Missing exports for Oak components
  - Accidental export of Internal components
  - Snapshots of unexpected components have been modified
  - Circular dependencies
  - Code duplication (via not using base components)
  - Non-functional storybook for this or related components
  - Sensitve files changed eg. atoms, or style tokens
  - Relative imports
  - Default exports

# Add your PR description below

-Fixes heading tag on container 
-Added expired lesson count
-Fixes li count for optionally cards

## A link to the component in the deployment preview

https://deploy-preview-267--lively-meringue-8ebd43.netlify.app/?path=/docs/components-organisms-teacher-oakunitscontainer--docs

### CODE REVIEW

**Review by RBrownsett — APPROVED** [review]
Approved - heading and tabindex accessibility issue fixed

**Review by k-huggs — CHANGES_REQUESTED** [review]

File: src/components/organisms/teacher/OakUnitListItem/OakUnitListItem.tsx [hunk 1]
```diff
@@ -100,6 +102,25 @@ export const OakUnitListItem = (props: OakUnitListItemProps) => {
...
+        ? lessonCount - expiredLessonCount
+        : null;
+    unitLessonCount =
+      expiredLessonsInUnitCount !== null
+        ? `${expiredLessonsInUnitCount}/${lessonCount} ${
+            lessonCount > 1 ? "lessons" : "lesson"
+          }`
+        : `${lessonCount} ${lessonCount > 1 ? "lessons" : "lesson"}`;
+
+    if (expiredLessonCount && expiredLessonCount > lessonCount) {
```
[the remark this answers is not in the document]
Comment (JBR90) [reply src/components/organisms/teacher/OakUnitListItem/OakUnitListItem.tsx:119]: To check is not null and it's greater than lesson count.

[the remark this answers is not in the document]
Comment (k-huggs) [reply src/components/organisms/teacher/OakUnitListItem/OakUnitListItem.tsx:119]: Apologies tired review yest evening

[the remark this answers is not in the document]
Comment (JBR90) [reply src/components/organisms/teacher/OakUnitListItem/OakUnitListItem.tsx:119]: No worries

**Review by k-huggs — APPROVED** [review]

**Review by k-huggs — APPROVED** [review]

**Pull request merged**